src/main/scala/com/azavea/modellab/Parser.scala

```diff
@@ -45,12 +48,35 @@ class Parser(layerRegistry: LayerRegistry, layerReader: FilteringLayerReader[Lay
   implicit class withJsonMethods(json: JsValue) {
     val fields = json.asJsObject.fields
     def get[T: JsonReader](name: String) = fields(name).convertTo[T]
-    def param[T: JsonReader](name: String) = 
+    def param[T: JsonReader](name: String): T =
       fields("parameters")
         .asJsObject
         .fields(name)
         .convertTo[T]
 
+    def optParam[T: JsonReader](name: String): Option[T] =
```

Comment:
Actually you should able to ask for `Option[T]` instead of `T` in params call like `params("neighborhood")[Option[Neighborhood]]`. Is that possible?